src/Microsoft.SqlTools.ServiceLayer/Migration/MigrationService.cs

```diff
@@ -625,18 +625,16 @@ internal static List<AzureSqlSkuCategory> GetEligibleSkuCategories(string target
                     string jsonFile = File.ReadAllText(Path.Combine(assemblyPath, RecommendationConstants.DataFolder, RecommendationConstants.SqlVmCapability));
                     List<AzureSqlIaaSCapability> vmCapabilities = JsonConvert.DeserializeObject<List<AzureSqlIaaSCapability>>(jsonFile);
 
-                    if (includePreviewSkus)
+                    // Eb series capabilities stored separately 
+                    string computePreviewFilePath = Path.Combine(assemblyPath, RecommendationConstants.DataFolder, RecommendationConstants.SqlVmPreviewCapability);
```

Comment:
nit: may be review 'preview' from variable name?

Reply:
just realized it's coming from a separate preview file :)... do we have plans to merge it. no harm in keeping as is. just curious.